### DISCUSSION TITLE
cannot input my fasta file

### DISCUSSION THREAD

**Discussion by tasos2310** [vdemichev/DiaNN#374] [General]
Hello ,
I am intrested for prediction of peptides of a list o isoforms that I have. I want a prediction with simple trypsin digest. I try to upload a simple fasta file with the peptides but seems that the system doesn't recognise the file. 
Could you please give me any idea about the solution?
Thank you in advance for your help.
Best regards,
Samaras Tasos

**Comment by vdemichev** [comment]
Hi Samaras,

Please share the log file. In general, FASTA files in UniProt format are supported, everything else is not guaranteed to work.

Best,
Vadim

**Comment by tasos2310** [comment]
Anastasios Samaras has shared a OneDrive for Business file with you. To view it, click the link below.
<https://ucdavis365-my.sharepoint.com/:u:/g/personal/asamaras_ucdavis_edu/EQvOTor2ufFEsEEnyh3bejABpw8Ep1fvYxKvdti86bl7XA>
[https://r1.res.office365.com/owa/prem/images/dc-generic_20.png]<https://ucdavis365-my.sharepoint.com/:u:/g/personal/asamaras_ucdavis_edu/EQvOTor2ufFEsEEnyh3bejABpw8Ep1fvYxKvdti86bl7XA>
UP000004994_4081.fasta (2).gz<https://ucdavis365-my.sharepoint.com/:u:/g/personal/asamaras_ucdavis_edu/EQvOTor2ufFEsEEnyh3bejABpw8Ep1fvYxKvdti86bl7XA>

Hi Vadim,
Thank you so much for your response. I try to upload 2 types of fasta files, one of this is from uniprot nut I still have problem.
Thank you in advance for your help.

Best,
Tasos
________________________________

**Comment by vdemichev** [comment]
Can you please attach / send me by email (1) the DIA-NN log file and (2) the FASTA file (need only a single one) which fails & indicating how this failure manifest (i.e. how you see that things don't work as expected).

**Comment by tasos2310** [comment]
Hi Vadim,
Sorry for my late response. I solve the problem the previous time, was my fault in the file that I searched for the results. Now I have already raw data and I am trying to analyze it using DIA-NN. More tutorial videos for data analysis like plexDIA, will be extremely helpful.

Thank you,
Best regards,
Samaras Anastasios
________________________________

**Comment by tasos2310** [comment]
Finally your previous email went to spam😁
________________________________
